package.json

```diff
@@ -50,7 +49,8 @@
     "build": "rm -rf build; babel src --out-dir build --presets=module:metro-react-native-babel-preset  --ignore 'src/__tests__/*'"
   },
   "jest": {
-    "preset": "react-native"
+    "preset": "react-native",
+    "moduleFileExtensions": ["js", "json"]
```

Comment:
we have `index.test.tsx` file that's now recognized by Jest 24, so I edited `moduleFileExtensions`.